README.md

```diff
@@ -91,15 +89,58 @@ val forecast: IO[Either[WeatherError, Current]] = client.forecastById(3070325, c
 
 will still be executed and OpenWeatherMap will decide how to handle it (in this case, it will ignore negative count).
 
+The caching client is created like this:
+```scala
+val cachingClient = OpenWeatherMap.cacheClient[IO]("YOUR_KEY")
+```
+More about caching later.
+
+### Dark Sky
+
+Sign up **[here][darkskydev]** to receive the API key. Dark Sky currently allows for 1000 free requests per day.
+
+#### Usage
+
+Similar to OpenWeatherMap, to create a basic client you can use the factories in the `DarkSky` object:
+```scala
+import com.snowplowanalytics.weather.providers.darksky.DarkSky
+import cats.effect.IO
+
+val client = DarkSky.basicClient[IO]("YOUR_KEY")
+```
+
+Dark Sky API is much simpler than OWM, it consists only of two functions, namely `forecast` and `timeMachine`
+
+- `forecast` returns the current weather and the forecast for next week
+- `timeMachine` returns the observed or forecast weather for the specified date in the past or the future
+
+Example:
+
+```scala
+import java.time.ZonedDateTime
+// Fetches weather a year ago in New York
+val response: IO[Either[WeatherError, DarkSkyResponse]] = 
```

Comment:
i'd respecify the `WeatherError` and add a `DarkSkyResponse` imports since people are most likely not gonna use both